frontend/src/components/ProjectSelector.tsx

```
@@ -0,0 +1,106 @@
+                        onChange={e => onSelect(e.target.value)}
+                    >
+                        <option value="">General Chat</option>
+                        {projects.map((p) => (
+                            <option key={p.canonical_github_url} value={p.canonical_github_url}>
+                                {p.repo_name}
+                            </option>
+                        ))}
+                    </select>
+                    {/* Custom dropdown arrow */}
+                    <div className="pointer-events-none absolute inset-y-0 right-2 flex items-center">
+                        <svg className="w-3 h-3 text-gray-400" fill="none" stroke="currentColor" strokeWidth="2" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)